src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -169,8 +169,12 @@ DistributedTableSize(Oid relationId, char *sizeQuery)
 							   " blocks which contain multi-shard data modifications")));
 	}
 
-	/* try to open relation, will error out if the relation does not exist */
-	relation = relation_open(relationId, AccessShareLock);
+	relation = try_relation_open(relationId, AccessShareLock);
+
+	if (relation == NULL)
+	{
+		ereport(ERROR, (errmsg("can't find table size, relation does not exist")));
```

Comment:
I know we (probably?) don't expect to hit this often but this message has several problems re: style.

  * _can't_ should be _cannot_ (avoid contractions)
  * Even _cannot_ should probably be _could not_, to show that this might be a temporary inability (if the table were recreated, the same invocation might work); PostgreSQL's original message here used _could not_ and I don't see a strong reason to change
  * the comma is a splice

Can we change this to: _could not compute table size: relation does not exist_?